pkgs/tools/X11/xtrace/default.nix

```diff
@@ -1,28 +1,23 @@
-{ stdenv, autoreconfHook, fetchgit, libX11, xauth, makeWrapper }:
+{ stdenv, autoreconfHook, fetchurl, libX11, xauth, makeWrapper }:
 
-let version = "1.3.1"; in
-stdenv.mkDerivation {
+let version = "1.4.0";
+in stdenv.mkDerivation {
   name = "xtrace-${version}";
-  src = fetchgit {
-    url = "git://git.debian.org/xtrace/xtrace.git";
-    rev = "refs/tags/xtrace-1.3.1";
-    sha256 = "1g26hr6rl7bbb9cwqk606nbbapslq3wnsy8j28azrgi8hgfqhjfi";
+  src = fetchurl {
```

Comment:
Please use `fetchFromGitLab` for downloading generated tarballs.

Reply:
Oh, we have that too now ! I wonder what is the point of turning a simple url into five different attributes, but... Here is the updated PR ;-).

PS: May I suggest https://github.com/NixOS/nixpkgs/pull/63596 in the meantime ?